docs/syntax_guide.rst

```diff
@@ -208,7 +208,7 @@ Additional specifiers for the :prop:`position` and :prop:`orientation` propertie
      - Positions the object uniformly at random in the given Region
    * - :sampref:`contained in {region}`
      - Positions the object uniformly at random entirely contained in the given Region
-   * - :sampref:`on {region}`
+   * - :sampref:`on ({region} | {Object} | {vector})`
      - Positions the base of the object uniformly at random in the given Region, or modifies the position so that the base is in the Region.
```

Comment:
Need to update the description too. Making the description short might be hard (since this needs to fit in a table without too much scrolling), so you could break it into two or more entries.